vtex/faststore
# Feat: Next 13  Root Layout

Depends on 
- https://github.com/vtex/faststore/pull/2298


## What's the purpose of this pull request?

This PR aims to create the root `layout.tsx` and bring the components from the `src` folder to the `app` folder.

Most of the changes from this PR are related to the path change to the `app` folder. But there are also new 'use clients' added to some components.

The sections should be implemented/moved in other PR's related [to another JIRA task](https://vtex-dev.atlassian.net/browse/SFS-880).

## How to test it?

Make sure that the older pages are working as expected and also the `http://localhost:3000/fs-next-update` is rendering (blank) without errors.

## Starter PR
TBD

## References

https://nextjs.org/docs/app/api-reference/file-conventions/layout
